ami/users/api/serializers.py

```diff
@@ -10,7 +10,7 @@
 class UserListSerializer(DefaultSerializer):
     class Meta:
         model = User
-        fields = ["id", "name", "details", "image"]
+        fields = ["id", "name", "details", "image", "email"]
```

Comment:
Note: I added email here because I needed it for the teams table, however I think this might also effect other API endpoints where we don't want to expose this in the response